rackspace/pom.xml

```diff
@@ -52,10 +52,9 @@
       <version>${jclouds.version}</version>
     </dependency>
     <dependency>
-       <groupId>org.apache.jclouds.labs</groupId>
+       <groupId>org.apache.jclouds.api</groupId>
        <artifactId>docker</artifactId>
-       <!-- TODO: Update version to variable jclouds.version on next release -->
-       <version>2.0.0-SNAPSHOT</version>
+       <version>${jclouds.version}</version>
```

Comment:
The code compiles and if it was working with 2.0.0-SNAPSHOT it should work with the 2.0.0 release. I think it is a safe change to make.